bootstrap/events/producer/streams_test.go

```diff
@@ -162,41 +175,52 @@ func TestAdd(t *testing.T) {
 
 		test(t, tc.event, event, tc.desc)
 
-		repoCall.Unset()
-		repoCall1.Unset()
-		repoCall2.Unset()
-		repoCall3.Unset()
-		repoCall4.Unset()
+		authCall.Unset()
+		authCall1.Unset()
+		sdkCall.Unset()
+		sdkCall1.Unset()
+		svcCall.Unset()
+		svcCall1.Unset()
 	}
 }
 
 func TestView(t *testing.T) {
```

Comment:
What are we testing here, because we don't check the emitted event is what we expect?